src/wizard/flutter/index.md

```diff
@@ -89,3 +89,16 @@ Future<void> processOrderBatch(ISentrySpan span) async {
 ```
 
 Check out [the documentation](https://docs.sentry.io/platforms/flutter/performance/instrumentation/) to learn more about the API and automatic instrumentations.
+
+## Uploads Debug Symbols
+
+Use the Sentry Dart Plugin to upload debug symbols for Android, iOS/macOS and source maps for Web to Sentry via [sentry-cli](https://docs.sentry.io/product/cli/).
```

Comment:
Lets instead link to the debug symbols page, the symbols page fallback to sentry-cli if people wanna do it manually.

Reply:
Let's also repeat the sentence that describes why the debug symbols are important, we can get from https://docs.sentry.io/platforms/flutter/upload-debug/ so people are more willing to install it.